realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -881,6 +881,108 @@ public void contains_caseSensitiveWithNonLatinCharacters() {
     }
 
     @Test
+    public void like_caseSensitive() {
+        final int TEST_OBJECTS_COUNT = 200;
+        populateTestRealm(realm, TEST_OBJECTS_COUNT);
+
+        RealmResults<AllTypes> resultList = realm.where(AllTypes.class)
+                .like("columnString", "*DaTa*").findAll();
```

Comment:
Gottcha.  What is the right margin supposed to be?  My right margin is at 100 and  .like clause doesn't fit.  That's why I wrapped it.